### ISSUE TITLE
Demo information not correct for some `NOA` records

### ISSUE THREAD

**Issue by nemecek-filip** [federicocalendino/nintendeals#26]
Hello,

I discovered that some games, like [Samurai Warriors 5](https://www.nintendo.com/us/store/products/samurai-warriors-5-switch/) (NSUID: 70010000037933) have demo available on the NOA eShop, but the `game_info` returns that demo is not available. 

After looking into it, I found the cause in the `build_game` method which checks whether "Demo Available" is in `filters`, however some games like the Samurai Warriors 5 have "Demo **a**vailable" feature so the check doesn't work.

I solved it locally by doing this:

```python
game.features = {
        Features.DEMO: "Demo Available" in filters or "Demo available" in filters,
        Features.GAME_VOUCHER: "Game Voucher eligible" in filters,
        Features.DLC: extra.get("dlc", False) if extra else None,
        Features.ONLINE_PLAY: "Online Play" in nso_features,
        Features.SAVE_DATA_CLOUD: "Save Data Cloud" in nso_features,
    }
```

If this is an acceptable solution I am happy to open PR with the fix.

**Comment by nemecek-filip** [comment]
@fedecalendino Hi, think you'd find a minute to check this out? Thanks

**Comment by fedecalendino** [comment]
Hi! I believe the issue should be fixed in `v3.1.3` that I just released.

Also, I'll have to archive this repository as I don't have much time to support it.

Thank you for all the help,
Fede.

ps: feel free to fork it and release new versions of it on pypi under a new name.
